benchmarks/benchmark_sufficient_stats_vs_groupby.R

```diff
@@ -0,0 +1,308 @@
+# This file compares the performance of the new Mundlak/sufficient statistics
```

Comment:
I'd like to tighten up this sim a bit (when I get a sec). But regardless of that, we should add this `benchmarks` folder to .Rbuildignore.